scripts/devnet-curio/curio.dockerfile

```diff
@@ -0,0 +1,102 @@
+#####################################
```

Comment:
Is this built on every PR and subsequent commits? If so, it'd be better to save it, the same way we do with the Lotus Dockefiles.

Reply:
I'll change it to build on demand.